docs/concepts/security.rst

```diff
@@ -13,7 +13,7 @@ This ensures that the control service, convergence agents, and API end users are
 Mutual Authentication Overview
 ==============================
 
-When :doc:`installing the flocker-node package <installing/index>`, the ``flocker-ca`` tool (provided as part of ``flocker-cli``) generates a root certificate for your cluster, comprising a private key and public certificate, as well as certificates and private keys for the control service and convergence agents.
+When :ref:`installing the flocker-node package <installflocker>`, the ``flocker-ca`` tool (provided as part of ``flocker-cli``) generates a root certificate for your cluster, comprising a private key and public certificate, as well as certificates and private keys for the control service and convergence agents.
```

Comment:
thanks @adamtheturtle - I'm reluctant to make changes to ref tags at the moment due to the risk of breaking links in other docs. I've raised FLOC 2262 to revisit this across the whole doc set post v1.